### ISSUE TITLE
SendRconCommand with console variables

### ISSUE THREAD

**Issue by NexiusTailer** [openmultiplayer/open.mp#608]
If using SendRconCommand from a script to switch the value of some of the console variables, omp server produces `Unknown command or variable:` message on it, no matter if you trying to do this with legacy name or updated.

Here is some example with different ways, all of them ends the same:
1. Using `SendRconCommand("query 0");` (just in samp manner without any changes)
2. Using `SendRconCommand("enable_query 0");` (name was changed, the type remains as in samp)
3. Using `SendRconCommand("enable_query false");` (name was changed, trying to treat it as bool as well)

Every of these variants gives the same runtime warning used inside OnGameModeInit callback:
```pawn
public OnGameModeInit()
{
	SendRconCommand("query 0");
	// or
	SendRconCommand("enable_query 0");
	// or
	SendRconCommand("enable_query false");
	return 1;
}
```

**Comment by NexiusTailer** [comment]
The same with typing var name into the server console manually with some value, example:

Console input: `varlist`
Console output: `<...> enable_query = 1 (bool)`
Console input: `query 0`
Console output: `Unknown command or variable: query 0`

Console input: `varlist`
Console output: `<...> enable_query = 1 (bool)`
Console input: `enable_query 0`
Console output: `Unknown command or variable: enable_query 0`

Console input: `varlist`
Console output: `<...> enable_query = 1 (bool)`
Console input: `enable_query false`
Console output: `Unknown command or variable: enable_query false`

Console input: `varlist`
Console output: `<...> enable_query = 1 (bool)`

---

SA-MP server won't refuse changing console var values both by SendRconCommand and manually by console inputs, if those variables are not read-only ofc. The ones which I picked for test fit the requirements to be able to be changed.

**Comment by Hual** [comment]
This should be fixed by #683
